packages/SwingSet/test/device-plugin/pingpong.js

```diff
@@ -0,0 +1,14 @@
+/* global harden */
+
+export function bootPlugin() {
+  return harden({
+    start(opts) {
+      const { prefix } = opts;
+      return harden({
+        ping(msg) {
```

Comment:
To test the transcript, this should pull `testLog` from the object passed as the first argument to `bootPlugin` (maybe call it `pluginPowers` to mimic `vatPowers`), and do `testLog('ping')` or something.

We're expecting `ping` to appear exactly once, even though the swingset has been reloaded from saved state. The bootstrap function is executed multiple times (once initially, then once per reload as the transcript is replayed), but the syscall it makes to talk to the device is simulated on the non-initial passes, so the device should only receive the initial call.